packages/create-sitecore-jss/src/templates/nextjs-personalize/src/components/CdpIntegrationScript.tsx

```diff
@@ -34,8 +34,6 @@ const CdpIntegrationScript = ({
   route: { itemLanguage, name },
   pageEditing,
 }: CdpIntegrationProps): JSX.Element => {
```

Comment:
We should switch to using `useSitecoreContext` hook to access context values (`pageEditing`, `route`) so that we are decoupled from `Layout.tsx` prop injection.

Reply:
This will now be required with [this PR change (got rid of nextjs-personalize `Layout.tsx`)](https://github.com/Sitecore/jss/pull/980/files#diff-0dae17efb21ff6cfc0b89abf0a6bf169fecc27e153cd7318d2de05d1dd2ad27c).

Reply:
done now - using sitecoreContext instead + removed extra class and params